cmake/autocoder/fpp.cmake

```diff
@@ -194,6 +194,10 @@ function(fpp_setup_autocode AC_INPUT_FILES)
         message(FATAL_ERROR "fpp tools not found, please install them onto your system path")
     endif()
     fpp_info("${AC_INPUT_FILES}")
+    set(CMAKE_BINARY_DIR_RESOLVED "${CMAKE_BINARY_DIR}")
+    set(CMAKE_CURRENT_BINARY_DIR_RESOLVED "${CMAKE_CURRENT_BINARY_DIR}")
+    resolve_path_variables(
+            AC_INPUT_FILES FPRIME_BUILD_LOCATIONS FPP_IMPORTS CMAKE_BINARY_DIR_RESOLVED CMAKE_BINARY_DIR_RESOLVED)
```

Comment:
Last 2 arguments `CMAKE_BINARY_DIR_RESOLVED CMAKE_BINARY_DIR_RESOLVED`
should probably be
`CMAKE_BINARY_DIR_RESOLVED CMAKE_CURRENT_BINARY_DIR_RESOLVED` (adding _CURRENT_)
no?

Reply:
Got catch, I will fix this!

Reply:
Fixed.